pkg/scaffold/rolebinding.go

```diff
@@ -54,7 +52,7 @@ subjects:
   name: default
   namespace: default
```

Comment:
Remove the ServiceAccount namespace. 
A Role binds to ServiceAccounts in the namespace it's created in.

Reply:
cool. Didn't know about that.

Reply:
fixed!